Src/Base/AMReX_IntVect.H

```diff
@@ -135,6 +135,15 @@ public:
 #endif
     }
 
+    template< typename T = int >
+    AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE
+    Array<T, AMREX_SPACEDIM>
+    toArray () const noexcept {
+        return Array<T, AMREX_SPACEDIM>{
```

Comment:
`Array` is `std::array` that does not work on device for C++11.

Reply:
Oh, thanks for the hint!
Would guarding this method with `__cplusplus` for C++14 be sufficient?